### PR TITLE
Settings Calendar create, view, edit, delete events

### PR DESCRIPTION
**Pull request by artem-blazhko**
## Description
Cover Settings Calendar functionality by e2e tests

## Links
- [Story - FAT-869](https://issues.folio.org/browse/FAT-869)
- [TestRail description](https://foliotest.testrail.io/index.php?/cases/view/347825)

## ScreenShots

Successfully passed test locally:

image

### CODE REVIEW

File: cypress/support/fragments/settings/calendar/library-hours/library-hours.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      .do(Button(buttonLabels.saveAndClose).click())
+      .get(selectors.eventInfo).contains(initialEventName);
+  },
+
+  editCalendarEvent() {
+    cy
+      .do(IconButton({ icon: iconsSet.edit }).click())
+      .expect(PaneHeader(pageHeaders.modifyEvent).exists());
+
+    cy.do(TextField({ name: nameAttributeValue }).fillIn(editedEventName))
```
Comment (Stanislau-Snapok) [inline cypress/support/fragments/settings/calendar/library-hours/library-hours.js:74]: Should we move `.do` to the next string as everywhere?

File: cypress/support/fragments/settings/calendar/library-hours/library-hours.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      .get(selectors.eventInfo)
+      .contains(editedEventName);
+  },
+
+  deleteCalendarEvent() {
+    cy
+      .do(IconButton({ icon: iconsSet.edit }).click())
+      .expect(PaneHeader(pageHeaders.modifyEvent).exists());
+
+    cy.do(Button(buttonLabels.delete).click())
```
Comment (Stanislau-Snapok) [inline cypress/support/fragments/settings/calendar/library-hours/library-hours.js:85]: The same as above.

**Review by Stanislau-Snapok — APPROVED** [review]

File: cypress/support/fragments/settings/calendar/library-hours/library-hours.js (hunk 2 above)
Comment (artem-blazhko) [reply cypress/support/fragments/settings/calendar/library-hours/library-hours.js]: done

File: cypress/support/fragments/settings/calendar/library-hours/library-hours.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+const fieldLabels = {
+  from: 'Valid From:*',
+  to: 'Valid To:*',
+};
+
+export default {
+  createCalendarEvent() {
+    cy
+      .visit(settingsMenu.calendarLibraryHoursPath)
+      .get(selectors.servicePoint).first().click()
```
Comment (IhorBohdan) [inline cypress/support/fragments/settings/calendar/library-hours/library-hours.js:47]: You can use here NavListItem interacrtor

Comment (artem-blazhko) [reply cypress/support/fragments/settings/calendar/library-hours/library-hours.js:47]: @IhorBohdan I need to find the first list item in the list. And should not rely on `label` of that item. And in this case, I can not use `NavListItem` interactor

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**